backend/src/game/newcards/discard_cards/victory_point_manipulation_cards/apartment_building.py

```diff
@@ -0,0 +1,10 @@
+from backend.src.game.newcards.discard_card import DiscardCard
+from backend.src.game.newcards.new_card import NewCard
+
+
+class ApartmentBuilding(DiscardCard):
+    def __init__(self):
+        NewCard.__init__(self, "Apartment Building", 5, "+ 3[Star]", "")
```

Comment:
This line and all others without footnotes could then drop the final argument in the init as it would be no longer necessary and would lead to cleaner code overall